package.json

```diff
@@ -10,5 +10,6 @@
   "license": "ISC",
   "devDependencies": {
     "eslint": "^7.22.0"
```

Comment:
I was having difficulties trying to get eslint working with the "type": "module" setting being set.
- Will try to figure out the reasoning behind this for the upcoming challenges.